### ISSUE TITLE
Children of bone attachments get deleted after re-importing .blend file

### ISSUE THREAD

**Issue by jitspoe** [godotengine/godot#73474]
### Godot version

4.0 RC 2

### System information

Windows 10, Forward+

### Issue description

If you have a bone attachment as a child of a skeleton of a blend file that's part of a scene ("editable children" set), modifying the .blend file will reimport it and delete the bone attachment children.

### Steps to reproduce

1. Create a scene (I used the CharacterBody3D as the root, but I don't know that it's a requirement).
2. Add a .blend file that contains a skeleton to the scene.
3. Check "Editable Children" on the blend file within the scene.
4. Add a bone attachment to the skeleton.
5. Add another node or scene under the bone attachment.
6. Leave the scene open and change the .blend file (ex: alter an animation and save).
7. Note that the children of the bone attachment no longer show up in the scene tree (though it seems they still show up in 3D).
8. Save the scene and the deletion will be permanent.

### Minimal reproduction project

[test_reimport_deletes_nodes.zip](https://github.com/godotengine/godot/files/10762573/test_reimport_deletes_nodes.zip)

**Comment by akien-mga** [comment]
CC @godotengine/import

**Comment by lyuma** [comment]
I can't reproduce this, testing 4.1-stable right now.

In my case, I'm hitting the Reimport button instead of editing it in Blender. I also tried saving it from blender instead but it also did not reproduce it.

Here is an example of the BoneAttachment3D and child nodes I added before the reimport / resave
![image](https://github.com/godotengine/godot/assets/39946030/7c79a116-e44b-4752-b918-b625b23bbcf6)

I tried an unsaved scene as well as a saved scene.

If you're also no longer able to reproduce this, I'll close the issue.
